### PR TITLE
Done

### PR DESCRIPTION
**Pull request by harsh-S21901**

### CODE REVIEW

File: App.API.Test/Controllers/AdminControllerTests/GetActiveUsersTest.cs
Comment (Prashant20985) [inline App.API.Test/Controllers/AdminControllerTests/GetActiveUsersTest.cs:1]: Add Tests for GetActiveUsers

File: App.API.Test/Controllers/AdminControllerTests/TestableAdminController.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace App.API.Test.Controllers.AdminControllerTests
+{
+    internal class TestableAdminController
+    {
```
Comment (Prashant20985) [inline App.API.Test/Controllers/AdminControllerTests/TestableAdminController.cs:10]: Add Expose Mediator method

**Review by Prashant20985 — CHANGES_REQUESTED** [review]

File: App.API.Test/Controllers/AdminControllerTests/GetActiveUsersTest.cs
Comment (harsh-S21901) [reply App.API.Test/Controllers/AdminControllerTests/GetActiveUsersTest.cs:1]: Resolved

File: App.API.Test/Controllers/AdminControllerTests/TestableAdminController.cs (hunk 1 above)
Comment (harsh-S21901) [reply App.API.Test/Controllers/AdminControllerTests/TestableAdminController.cs:9]: Resolved

**Review by Prashant20985 — APPROVED** [review]

**Pull request merged**